cobigen/index.asciidoc

```diff
@@ -1,8 +1,12 @@
-= Tutorial for Java Backend Generation with CobiGen
+= Generate your code with devonfw
```

Comment:
What to generate out of what? Is it a Java stub application out of nothing or services out of open api?

Reply:
Done